autotest/framework.py

```diff
@@ -46,6 +46,27 @@ def build_mf6_models(self, build_function, idx, exdir):
         if regression is not None:
             regression.write_simulation()
 
+    def build_mf6_models_legacy(self, build_function, idx, exdir):
```

Comment:
@jdhughes-usgs you mentioned that `write_input` is relevant for older versions of MODFLOW.
This is why I added the `legacy` suffix.
I am open to better naming suggestions